### DISCUSSION TITLE
Mixture of models in single dataset

### DISCUSSION THREAD

**Discussion by emilyjunkins** [adw96/breakaway#99] [Q&A]
Hi folks!

I've used `breakaway` on phyloseq object and wanted to check the model,
`ba$model`
but found it return
`NULL`

I wrote out a table and saw that my data are actually a mixture of models, mostly Neg Binomial, but some Kemp and few Poisson. Using `breakaway_nof1`, I found that it was mostly Poisson and Kemp. Assuming breakaway acts independently on each sample, is it suspicious that there are multiple models in a single dataset? Is there an interpretation to be had here regarding the data structure?

Also, does this affect downstream betta() ?

**Comment by adw96** [comment]
Great question and thanks for posting @emilyjunkins ! Indeed breakaway always estimates the richness of samples independently, and in fact it's pretty common to see different models chosen even in a single data set (not suspicious at all). It won't adversely impact betta downstream, either. 

Hope that helps and thanks for kickstarting our Discussions!
